HousingRepairsOnlineApi/Domain/RepairRequest.cs

```diff
@@ -3,6 +3,7 @@ namespace HousingRepairsOnlineApi.Domain
 {
     public class RepairRequest
     {
+        public string RepairType { get; set; } /// Communal, Tenant, Leasehold
```

Comment:
This comment about the potential values could become a disconnected from acceptable values and then are a source of disinformation. It would be best to remove the comment and rely on other documentation.